### PR TITLE
fix: race condition when applying namespace quotas

### PR DESCRIPTION
**Pull request by luisdavim**
this should fix: #588

### CODE REVIEW

File: internal/app/kube_helpers.go [hunk 1]
```diff
@@ -119,19 +119,10 @@ spec:
...
-	if err := ioutil.WriteFile(targetFile, []byte(definition), 0o666); err != nil {
-		log.Fatal(err.Error())
-	}
-
-	cmd := kubectl([]string{"apply", "-f", targetFile, "-n", ns, flags.getKubeDryRunFlag("apply")}, "Creating LimitRange in namespace [ "+ns+" ]")
-	result := cmd.Exec()
 
-	if result.code != 0 {
-		log.Fatal("Failed to create LimitRange in namespace [ " + ns + " ] with error: " + result.errors)
+	if err := apply(definition, ns, "LimitRange"); err == nil {
```
Comment (Revellski) [inline internal/app/kube_helpers.go:123]:
```suggestion
	if err := apply(definition, ns, "LimitRange"); err != nil {
```

File: internal/app/kube_helpers.go [hunk 2]
```diff
@@ -163,18 +154,31 @@ spec:
 
 	definition = definition + Indent(string(d), strings.Repeat(" ", 4))
 
-	if err := ioutil.WriteFile("temp-ResourceQuota.yaml", []byte(definition), 0o666); err != nil {
+	if err := apply(definition, ns, "ResourceQuota"); err == nil {
```
Comment (Revellski) [inline internal/app/kube_helpers.go:157]:
```suggestion
	if err := apply(definition, ns, "ResourceQuota"); err != nil {
```

**Review by Revellski — COMMENTED** [review]
```
2021-03-13 09:42:00 INFO: Creating ResourceQuota in namespace [ namespace-A ]
2021-03-13 09:42:00 DEBUG: kubectl apply -f .helmsman-tmp/ResourceQuota-075781935.yaml -n namespace-A
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xc46f98]
goroutine 67 [running]:
github.com/Praqma/helmsman/internal/app.setQuotas(0xc00050ef50, 0xf, 0xc000609810)
        /home/kyp/projects/helmsman/internal/app/kube_helpers.go:158 +0x3d8
github.com/Praqma/helmsman/internal/app.addNamespaces.func1(0xc00050ef50, 0xf, 0xc000534380, 0xc000332010)
        /home/kyp/projects/helmsman/internal/app/kube_helpers.go:31 +0x135
created by github.com/Praqma/helmsman/internal/app.addNamespaces
        /home/kyp/projects/helmsman/internal/app/kube_helpers.go:24 +0x12f
```

**Comment by luisdavim** [comment]
Oh, sorry about that. I guess that's what you get when opening PRs past bed time 🤦

**Pull request merged**